framework/doc/content/source/meshmodifiers/TimedSubdomainModifier.md

```diff
@@ -9,7 +9,7 @@ For geometrically complex models, an efficient option is to define the volumes t
 Addressing this task, the `TimedSubdomainModifier` eases re-assignment of all elements of a subdomain, makes Moose input files shorter, and reduces
 potential sources of error compared to the usage of [CoupledVarThresholdElementSubdomainModifier.md] (which in turn has its advantages if the geometry is not organized into subdomains).
 
-The `TimedSubdomainModifier` inherits from the [CoupledVarThresholdElementSubdomainModifier.md]. Details on solution initialization, stateful material property initialization and moving boundary/interface nodeset/sideset modification can be found in the description of the [CoupledVarThresholdElementSubdomainModifier.md].
+The `TimedSubdomainModifier` inherits from the [ElementSubdomainModifier.md]. Details on solution reinitialization, stateful material property reinitialization and moving boundary/interface nodeset/sideset modification can be found in the description of the [ElementSubdomainModifier.md].
```

Comment:
off topic, but I m curious if the nodeset modification works with TimedSubdomainMod
there isnt as much of a front iirc, an entire subdomain can get swapped

Reply:
There is an example in the tests called 'return_moving_boundary', which has a boundary defined between subdomains 1 and 2. It uses the `TimedSubdomainModifier` to change subdomain 1 to 3, and then back, and the nodeset reflects that change.

Reply:
great!